src/ProductConstructionService/ProductConstructionService.DependencyFlow/PullRequestBuilder.cs

```diff
@@ -351,6 +362,14 @@ private static string GenerateUpstreamRepoDiffs(IReadOnlyCollection<UpstreamRepo
         return sb.ToString();
     }
 
+    private static string GenerateDarcDiffHelp(BuildDTO build, string targetRepository, string headBranch) =>
+        $"""
+        <!--
+            To diff the source repo and PR branch contents locally, run:
```

Comment:
Is it fair to say that Maestro PRs are the first place where many devs would hear about darc? If so, how about adding a link to the setup/usage guide somewhere?
```suggestion
            To diff the source repo and PR branch contents using the [DARC tool](https://github.com/dotnet/arcade-services/blob/main/docs/Darc.md), run:
```